aidial_interceptors_sdk/examples/chat_completion/replicator.py

```diff
@@ -181,3 +192,13 @@ async def on_stream_end(self) -> None:
             else FinishReason(self.finish_reasons[1])
         )
         self.send_chunk(EndChoiceChunk(finish_reason, 0))
+
+
+_T = TypeVar("_T")
+
+
+async def _join_iterators(iters: List[AsyncIterator[_T]]) -> AsyncIterator[_T]:
+    combine = aiostream.stream.merge(*iters)
+    # FIXME: UserWarning: Streamer is iterated outside of its context
```

Comment:
```python
async def _join_iterators(iters: List[AsyncIterator[_T]]) -> AsyncIterator[_T]:
    async with aiostream.stream.merge(*iters).stream() as combine:
        async for item in combine:
            yield item
```
This will remove warning

Reply:
fixed